WooCommerce/Classes/ViewRelated/Products/Variations/Bulk Update/BulkUpdateViewController.swift

```diff
@@ -0,0 +1,236 @@
+//
+private extension BulkUpdateViewController {
+    func rowAtIndexPath(_ indexPath: IndexPath) -> Row {
+        return sections[indexPath.section].rows[indexPath.row]
+    }
+}
+
+extension BulkUpdateViewController {
+    struct Section: Equatable {
+        let title: String?
+        let rows: [Row]
+    }
+
+    enum Row: CaseIterable {
+        case regularPrice
+
+        fileprivate var type: UITableViewCell.Type {
+            return ValueOneTableViewCell.self
+        }
+
+        fileprivate var reuseIdentifier: String {
+            return type.reuseIdentifier
+        }
+    }
+}
+
+private struct Constants {
+    static let sectionHeight = CGFloat(44)
+    static let placeholderRowsPerSection = [1]
+}
+
+private extension BulkUpdateViewController {
+    /// Renders the Placeholder Content.
+    ///
+    func displayGhostContent() {
+        guard let ghostView = ghostTableViewController.view else {
+            return
+        }
+
+        ghostView.translatesAutoresizingMaskIntoConstraints = false
+        addChild(ghostTableViewController)
+        view.addSubview(ghostView)
+        view.pinSubviewToAllEdges(ghostView)
+        ghostTableViewController.didMove(toParent: self)
+    }
+
+    /// Removes the Placeholder Content.
+    ///
+    func removeGhostContent() {
+        guard let ghostView = ghostTableViewController.view else {
+            return
+        }
+
+        ghostTableViewController.willMove(toParent: nil)
+        ghostView.removeFromSuperview()
+        ghostTableViewController.removeFromParent()
+    }
+
+    /// A  controller that is shown when `displayGhostContent()` is called. Added as a child view controller.
+    ///
+    final class GhostTableViewController: UITableViewController {
+
+        init() {
+            super.init(style: .plain)
+        }
+
+        required init?(coder: NSCoder) {
+            fatalError("init(coder:) has not been implemented")
+        }
+
+        override func viewDidLoad() {
+            super.viewDidLoad()
+
+            tableView.dataSource = nil
+            tableView.delegate = nil
+
+            tableView.backgroundColor = .listBackground
+            tableView.estimatedRowHeight = Constants.sectionHeight
+            tableView.applyFooterViewForHidingExtraRowPlaceholders()
+            tableView.registerNib(for: ValueOneTableViewCell.self)
+        }
+
+        /// Activate the ghost if this view is added to the parent.
+        ///
+        override func viewWillAppear(_ animated: Bool) {
+            super.viewWillAppear(animated)
+
+            let options = GhostOptions(reuseIdentifier: ValueOneTableViewCell.reuseIdentifier,
+                                       rowsPerSection: Constants.placeholderRowsPerSection)
+            tableView.displayGhostContent(options: options,
+                                          style: .wooDefaultGhostStyle)
+        }
+
+        /// Deactivate the ghost if this view is removed from the parent.
```

Comment:
I don't think we need to create a new type for this.
A simpler approach is to have a separate `UITableView` and just call `.displayGhostContent` on it.

Example: 
https://github.com/woocommerce/woocommerce-ios/blob/07fdba67367f3127bcbf16151737d9204d9ccc11/WooCommerce/Classes/ViewRelated/Dashboard/Settings/Plugins/PluginListViewController.swift#L14

Reply:
True initially I followed the solution from the issue that originally solved the race condition problem, I updated the PR to simply use a second tableview in the same view controller.